community.md

```diff
@@ -126,13 +126,18 @@ title: Trino Community
             </div>
             <div>
               <h3>Open a pull request</h3>
-              <p>Improve exisiting functionality or even add your own feature.</p>
+              <p>Improve existing functionality or even add your own feature.</p>
               <p>Read about our <a href="../development/process.html">contribution process</a> or visit our <a href="../docs/current/develop.html">developer guide</a> to learn more.</p>
             </div>
           </div>
         </div>
       </div>
     </div>
+    <div class="community-content-section">
+      <h2>Trino Contributors</h2>
+      <p>A huge amount of gratitude goes out to all of our contributors. Without you, none of this is possible.</p>
+        <object type="image/svg+xml" data="https://contrib.rocks/image?repo=trinodb/trino&max=200&columns=10"></object>
```

Comment:
Would it be safer to just use an image and update it now an then .. this would avoid the dependency on that system being up and running and available